### PR TITLE
Copy edits based on user testing and Kate's observations

### PR DESCRIPTION
**Pull request by cpapazian**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -33,6 +33,18 @@ en:
       invalid_email: You must enter the email address that matches this account.
       alert: 'This will delete all of your account data, applications, and history.  Proceed with caution!'
       apps_deletion_warning: 'The following applications will also be deleted:'
+  devise:
+    welcome:
+      line_1: Welcome to MyUSA from %{app_name}
+      line_2: Create a MyUSA account or log in to your existing account to access %{app_name}.
+    connect_with: Connect with MyUSA
+    enter_your_email:  Enter your email to log in
```
Comment (polastre) [inline config/locales/en.yml]: email or email address?

**Comment by polastre** [comment]
tagging @kategarklavs.

for @cpapazian -- if you're adding `devise` to the translations, should you also add the 2FA screen?

**Comment by cpapazian** [comment]
was just trying to come up with a naming convention for our edits.

File: config/locales/en.yml (hunk 1 above)
Comment (esgoodman) [inline config/locales/en.yml]: Not to be anal-retentive, but are we doing two spaces after periods, or one? (.e.g before "proceed with caution.")

File: config/locales/en.yml [hunk 2]
```diff
@@ -33,6 +33,18 @@ en:
...
+    almost_there: You're almost there
```
Comment (esgoodman) [inline config/locales/en.yml]: Does this need a period at the end of the line?

File: config/locales/en.yml [hunk 3]
```diff
@@ -33,6 +33,18 @@ en:
       invalid_email: You must enter the email address that matches this account.
       alert: 'This will delete all of your account data, applications, and history.  Proceed with caution!'
```
Comment (esgoodman) [inline config/locales/en.yml]: This is where I noticed the two spaces after a period.

File: config/locales/en.yml (hunk 2 above)
Comment (esgoodman) [inline config/locales/en.yml]: Oh, no, wait. I see. This is a heading. Sorry.

File: config/locales/en.yml [hunk 4]
```diff
@@ -33,6 +33,18 @@ en:
...
+    connect_with: Connect with MyUSA
+    enter_your_email:  Enter your email to log in
+    almost_there: You're almost there
+    email:
+      instructions:
+        email_sent: We just sent an access link to %{email}.
+        click_that_link: Click the link (or copy and paste it into your browser) to finish logging in.
+        new_link_request: 'If you don’t receive this email within the next five minutes, please check your spam folder. If you don’t find it there, you may <a href="%{request_link_link}">request another link</a>.'
   no_authorizations: You have not authorized any applications.
   no_user_token: There was a problem resending your token. Please try creating your account again.
```
Comment (esgoodman) [inline config/locales/en.yml]: Not sure when/where this text shows up, but "token" seems a little technical for user-facing copy. Is it equivalent to "access link," which is used elsewhere?

File: app/views/devise/sessions/_login.html.erb [hunk 5]
```diff
@@ -35,7 +35,7 @@
   <div class="row">
     <div class="col-md-12">
       <h1 class="text-center">
-        Connect to MyUSA
```
Comment (esgoodman) [inline app/views/devise/sessions/_login.html.erb]: The change to "Connect with" from "Connect to" is the main change drawing specifically on user research, not just copy editing best practices. See my comment in the conversation for more.

File: config/locales/en.yml (hunk 3 above)
Comment (kategarklavs) [inline config/locales/en.yml]: There should only be one space after a period!

**Comment by polastre** [comment]
@cpapazian my comment wasn't about the naming convention; it was about adding in the 2FA screen text if you're doing this work for the other screens.

File: config/locales/en.yml (hunk 4 above)
Comment (cpapazian) [inline config/locales/en.yml]: we aren't changing the token strings in this task. agree that those screens could use some better copy.

**Comment by cpapazian** [comment]
@polastre are you suggesting pulling other strings out of html into the translations as part of htis task? would rahter just do it as we make changes, rather than all at once.

@esgoodman @kategarklavs is there anything else we need to do here?

**Comment by polastre** [comment]
:+1: from me then, just wanted to be thorough with what we pull out into strings.

File: config/locales/en.yml (hunk 4 above)
Comment (esgoodman) [inline config/locales/en.yml]: As long as there's no reason why any user would actually see the no_user_token string, then I think we're good.

Comment (kategarklavs) [inline config/locales/en.yml]: Agree with @esgoodman (though I'm happy to work on token strings, if you all prefer).

**Pull request merged**